File: 02_fundamentals_python/12_functions/12_exercises/04_odd_and_even_sum.py
def sum_of_even_and_sum_of_odd(number):
    list_of_digits = []
    for i in range(len(number)):
        num = int(number[i])
        list_of_digits.append(num)
    odd_sum = 0
    even_sum = 0
    for element in list_of_digits:
        if int(element) % 2 == 0:
            even_sum += int(element)
        else:
            odd_sum += int(element)
    return f"Odd sum = {odd_sum}, Even sum = {even_sum}"


given_number = input()
print(sum_of_even_and_sum_of_odd(given_number))

File: 03_python_professional/04_js_front_end/02_Functions and Statements/02_exercises/04. Odd And Even Sum.js
function sum(num) {
  let nums = String(num).split("");
  let evenSum = 0;
  let oddSum = 0;
  for (let i = 0; i < nums.length; i++) {
    let currentNum = Number(nums[i]);
    if (currentNum % 2 === 0) {
      evenSum += currentNum;
    } else {
      oddSum += currentNum;
    }
  }
  let result = `Odd sum = ${oddSum}, Even sum = ${evenSum}`
  console.log(result)
}

sum(3495892137259234);
